Sources/Foundation/URL.swift

```diff
@@ -534,6 +535,11 @@ public struct URL : ReferenceConvertible, Equatable {
     public init(fileURLWithPath path: String) {
         _url = URL._converted(from: NSURL(fileURLWithPath: path.isEmpty ? "." : path))
     }
+#else
+    public init(fileURLWithPath path: String) {
+        _url = NSURL(string: "file://\(path)")!
```

Comment:
Also, this should at least check that path begins with a slash. Otherwise passing in "foo" results in the URL file://foo (i.e. with "foo" in the hostname position and no path)